scripts/xsslint/xsslint/linters.py

```diff
@@ -448,7 +448,7 @@ def _check_javascript_interpolate(self, file_contents, results):
 
         """
         # Ignores calls starting with "StringUtils.", because those are safe
-        regex = re.compile(r"(?<!StringUtils).interpolate\(")
+        regex = re.compile(r"(?<!StringUtils\.)interpolate\(")
```

Comment:
Is this supposed to be nearly equivalent, but just cleaner?  Or, is this meant to be a fix?  Could you update `test_javascript_interpolate` to clarify?

Reply:
This was a fix to get `test_javascript_interpolate` to pass. (that was the test that was previously broken, but shadowed by the next method having the same name.)